Test/WebTestCase.php

```diff
@@ -552,10 +552,10 @@ private function locateResources($paths)
      */
     public function loadFixtureFiles(array $paths = array(), $append = false, $omName = null, $registryName = 'doctrine', $purgeMode = null)
     {
-        if (!class_exists('Nelmio\Alice\Fixtures')) {
+        if (!class_exists('Nelmio\Alice\Fixtures') && !(class_exists('Nelmio\Alice\FixtureSet') && class_exists('Fidry\AliceDataFixtures\Loader\PersisterLoader'))) {
```

Comment:
Missing negative condition for `class_exists('Fidry\AliceDataFixtures\Loader\PersisterLoader')` AFAIK.

Reply:
I also think it would be safer to test against an interface rather than a specific class